src/server.c

```diff
@@ -2519,111 +2519,16 @@ void commandAddSubcommand(struct redisCommand *parent, struct redisCommand *subc
     serverAssert(dictAdd(parent->subcommands_dict, sdsnew(subcommand->name), subcommand) == DICT_OK);
 }
 
-
-/* Parse the flags string description 'strflags' and set them to the
- * command 'c'. Abort on error. */
-void parseCommandFlags(struct redisCommand *c, const char *strflags) {
-    int argc;
-    sds *argv;
-
-    /* Split the line into arguments for processing. */
-    argv = sdssplitargs(strflags,&argc);
-    if (argv == NULL)
-        serverPanic("Failed splitting strflags!");
-
-    for (int j = 0; j < argc; j++) {
-        char *flag = argv[j];
-        if (!strcasecmp(flag,"write")) {
-            c->flags |= CMD_WRITE|CMD_CATEGORY_WRITE;
-        } else if (!strcasecmp(flag,"readonly")) {
-            c->flags |= CMD_READONLY|CMD_CATEGORY_READ;
-        } else if (!strcasecmp(flag,"denyoom")) {
-            c->flags |= CMD_DENYOOM;
-        } else if (!strcasecmp(flag,"admin")) {
-            c->flags |= CMD_ADMIN|CMD_CATEGORY_ADMIN|CMD_CATEGORY_DANGEROUS;
-        } else if (!strcasecmp(flag,"pubsub")) {
-            c->flags |= CMD_PUBSUB|CMD_CATEGORY_PUBSUB;
```

Comment:
i think we still need this method to run at runtime and apply the implicit flags.
i.e. we removed the string compare, but we still need some runtime processing.
we could also do that in the python script, but i think it's better here.